Zend/zend_fibers.c

```diff
@@ -516,10 +526,7 @@ static zend_always_inline zend_fiber_transfer zend_fiber_suspend(zend_fiber *fib
 
 static zend_object *zend_fiber_object_create(zend_class_entry *ce)
 {
-	zend_fiber *fiber;
-
-	fiber = emalloc(sizeof(zend_fiber));
-	memset(fiber, 0, sizeof(zend_fiber));
+	zend_fiber *fiber = ecalloc(1, sizeof(zend_fiber));
```

Comment:
Please do not use ecalloc unless it has a dynamic argument.

Reply:
Reverted, but why is it preferred to use `emalloc()` + `memset()` over a single call to `ecalloc()` if you need zeroed memory?

Reply:
It's much slower. Doesn't particularly matter here, but emalloc+memset is our general preference.